packages/ui5-application-writer/src/index.ts

```diff
@@ -50,10 +50,20 @@ async function generate(basePath: string, ui5AppConfig: Ui5App, fs?: Editor): Pr
     // ui5-local.yaml
     const ui5LocalConfigPath = join(basePath, 'ui5-local.yaml');
     const ui5LocalConfig = await UI5Config.newInstance(fs.read(ui5LocalConfigPath));
+    const ui5LibsArray = ui5App.ui5.ui5Libs as string[];
+    //remove reuse libs
+    const newUI5LibsArray = [];
+    for (const ui5Lib of ui5LibsArray) {
+        const reuseLib = ui5Lib.includes('.lib.reuse');
```

Comment:
Is `.lib.reuse` standard and can be used to determine this?
Or should the additional libs from https://github.com/SAP/open-ux-tools/blob/e9a78ae5400045940a039d90edc496397cb64845/packages/fiori-elements-writer/src/data/annotationReuseLibs.ts#L6 have an additional property to indicate that they should not be added to the local yaml

Reply:
^ this is as much a question as a suggestion

Reply:
Thank you, I have changed it